Update Z-Wave documentation OpenZWave 1.6

pauldegraaff opened the pull request:
## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->
Informing users that there are two z-wave integrations available. While also noting that OpenZWave 1.4 is EOL. And that the new integration is based on OpenZWave 1.6

## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [ x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

pauldegraaff committed:
Update Z-Wave documentation OpenZWave 1.6

frenck commented on source/_docs/z-wave.markdown:
```diff
@@ -3,6 +3,17 @@ title: "Z-Wave"
 
+Home Assistant currently offers two Z-Wave integrations. 
+
+1. [Z-Wave](https://www.home-assistant.io/integrations/zwave/) based on OpenZWave 1.4 (EOL)
```
Our integration is not EOL.

pauldegraaff replied:
Correct, however OpenZWave 1.4 is EOL. That is what it reads.

frenck replied:
That is not what it reads, the subject of this sentence is the Z-Wave integration of Home Assistant.

pauldegraaff replied:
I clarified the subject in a new commit, so that it reads that OpenZWave 1.4 = EOL and not the integration itself.

pauldegraaff committed:
Clarified that OpenZWave 1.4 is EOL

Clarified that OpenZWave 1.4 is EOL not the integration by putting quotation marks around the subject.

frenck commented on source/_docs/z-wave.markdown:
```diff
@@ -3,6 +3,17 @@ title: "Z-Wave"
 
+Home Assistant currently offers two Z-Wave integrations. 
+
+1. [Z-Wave](https://www.home-assistant.io/integrations/zwave/) based on "OpenZWave 1.4 (EOL)"
+2. [OpenZWave (Beta)](https://www.home-assistant.io/integrations/ozw/) based on "OpenZWave 1.6"
+
+<div class='note'>
+  
+OpenZWave 1.4 is end-of-life. This means it is no longer maintained and your devices might not be supported. The [OpenZWave (Beta)](https://www.home-assistant.io/integrations/ozw/) integration will replace the [Z-Wave](https://www.home-assistant.io/integrations/zwave/) integration.
```
Don't use full-blown URLs

frenck commented:
I'm not sure about this addition and the reason why the deprecation is so hard. I'm not sure what EOL means in this context myself (even with your added notes). The 1.4 database hasn't been maintained for ages, that isn't new or changed.

Basically, our Z-Wave integration is supported and actually recommended.

pauldegraaff commented:
> 
> 
> I'm not sure about this addition and the reason why the deprecation is so hard. I'm not sure what EOL means in this context myself (even with your added notes). The 1.4 database hasn't been maintained for ages, that isn't new or changed.
> 
> Basically, our Z-Wave integration is supported and actually recommended.
> 
> I'm fine with offering options. I'm not fine with the text offering no good option.

It is all factual information. There is no subjectiveness in pushing users to either one of the integration options. However with this information, users can make there own informed decision. Because like you said, OpenZWave 1.4 is very much outdated.

frenck commented:
The fact is, we support the z-wave integration, and currently, OZW is in beta.

The current text is putting users in limbo, which is not guiding, but causing deliberate confusion.
I'm not willing to merge it in the current state.

frenck commented:
As per discussion on Discord, closing this PR.
Will look into this myself tomorrow.
